actions/drain_node_handler.go

```diff
@@ -54,7 +54,7 @@ func (h *drainNodeHandler) Handle(ctx context.Context, data interface{}) error {
 		return fmt.Errorf("unexpected type %T for delete drain handler", data)
 	}
 
-	log := h.log.WithField("node_name", req.NodeName)
```

Comment:
This is a drain handler, please consider removing 'delete' from the logging message above.